style.css

```diff
@@ -1016,6 +1025,11 @@ input[type="submit"]:disabled {
   }
 
   .modal-box {
+    width: 77%;
+    height: auto;
+    background-color: #fff;
+    padding: 20px;
+    position: relative;
     margin: 10%;
```

Comment:
- [OPTIONAL ] :thinking: I think it would be better if your design was responsive enough to keep the popup in the middle of the screen as per the Figma design. For that I think you should check well the padding you added again.

__Please note that this change is optional only because the previous reviewer didn't mention it__

Your design:
![yourDetails](https://user-images.githubusercontent.com/97834160/209142656-9256f052-f04c-4478-adec-080cd9a406da.png)
Figma design:
![DetailsPopup](https://user-images.githubusercontent.com/97834160/209142729-da77e4aa-884f-45bb-80c1-1d8a12baf9a9.png)